content/acra/configuring-maintaining/general-configuration/acra-keys/destroy.md

```diff
@@ -389,7 +386,13 @@ Currently, only some key kinds are supported for destroying via `destroy` subcom
 Here is the list of supported key kinds:
```

Comment:
> **Note:**
> Currently, only some key kinds are supported for destroying via `destroy` subcommand.

I'm not sure we should leave this note anymore. Previously acra-keys destroy supported subset of kind of keys. Now it supports all available kinds.